os/BUILD

```diff
@@ -111,3 +111,14 @@ constraint_value(
     name = "chromiumos",
     constraint_setting = ":os",
 )
+
+constraint_value(
+    name = "trusty",
+    constraint_setting = ":os",
+)
+
+constraint_value(
```

Comment:
Is ayslo really an OS?  Or is it really just a hypervisor API that can be melded with any OS.  That is, can we build for ayslo framework compliant posix enclave and also have a choice of an ayslo compliant windows enclave.

Again, if only Google needs this, we should keep it private.